OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -224,3 +306,14 @@ def toggleTransducerTrackingApproval(self) -> None:
             raise RuntimeError("Cannot toggle tracking approval because there is no active session.")
         session.toggle_transducer_tracking_approval() # apply the approval or lack thereof
         data_parameter_node.loaded_session = session # remember to write the updated session object into the parameter node
+
+    def runTransducerTracking(self,
+                              inputProtocol: SlicerOpenLIFUProtocol,
+                              inputTransducer : SlicerOpenLIFUTransducer,
+                              inputVolume: vtkMRMLScalarVolumeNode,
+                              inputPhotoscan: vtkMRMLModelNode,
+                              inputTRS: vtkMRMLModelNode
+                              ) -> Tuple[vtkMRMLTransformNode, vtkMRMLTransformNode]:
+        ## Need to integrate with transducer tracking library here
+        print("Transducer tracking integration under development")
```

Comment:
Better to raise an infobox here, since we plan to merge this PR and it could end up in a release.